packages/flutter/lib/src/rendering/table.dart

```diff
@@ -1050,26 +1051,26 @@ class RenderTable extends RenderBox {
     if (rows * columns == 0) {
       // TODO(ianh): if columns is zero, this should be zero width
       // TODO(ianh): if columns is not zero, this should be based on the column width specifications
+      _tableWidth = 0.0;
```

Comment:
Does this resolve the TODOs?

Reply:
I am not quite understood what the todos mean, but I think maybe they can be removed.